content/en/lotus/manage/claim-extension.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Claim Extension"
+date: 2024-03-08T18:40:56+04:00
+description: "This page explains verified deal terminologies and how to extend a verified claim"
+menu:
+  lotus:
+    parent: "lotus-management"
+weight:
+toc: true
+draft: false
+---
+
+With the introduction of [FIP-0045: De-couple verified registry from markets](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0045.md), the terms of DataCap allocations are independent of market storage deals.
+The DataCap is now represented as a fungible token. These changes provide a foundation for user-programmed market actors to broker Filecoin Plus verified deals in the future.
+
+## What is an Allocation?
+An allocation specifies a range of terms for which the provider may commit the data, between some minimum and maximum. An allocation's maximum term must be at least as large as its minimum term.
+An allocation may be removed after its expiration epoch has passed (by anyone) and it was not converted to a claim. When removed, the DataCap tokens are transferred back to the client
+
+| Terminology                         | Definition                                                                                            |
+|-------------------------------------|-------------------------------------------------------------------------------------------------------|
+| MinimumVerifiedAllocationTerm       | The minimum allowed value for TermMinimum is 6 months                                                 |
+| MaximumVerifiedAllocationTerm       | The maximum allowed value for TermMaximum is 5 years (can be increased by a future FIP)               |
+| MaximumVerifiedAllocationExpiration |  The maximum difference between Expiration and the epoch at which an allocation is created is 60 days |
+
+{{< alert icon="tip" >}}
+Since, Datacap is independent of the time for which data is stored, it is recommended set TermMaximum as MaximumVerifiedAllocationTerm to store your data for maximum possible period.
```

Comment:
```suggestion
Since DataCap is independent of the time for which data is stored, it is recommended to set TermMaximum as MaximumVerifiedAllocationTerm to store your data for the maximum possible period.
```